src/generateTypeFiles.js

```diff
@@ -76,15 +77,18 @@ export const generateTypeFileContents = (
             (options.regenerateCommand
                 ? `// To regenerate, run '${options.regenerateCommand}'.\n`
                 : '') +
-            code.replace(/\s+$/g, '');
+            code;
         if (options.splitTypes && !isFragment) {
             fileContents +=
                 `\nexport type ${name} = ${typeName}['response'];\n` +
                 `export type ${name}Variables = ${typeName}['variables'];\n`;
         }
+        Object.keys(extraTypes).forEach((name) => {
+            fileContents += `\n\nexport type ${name} = ${extraTypes[name]};`;
+        });
 
         addToIndex(targetPath, typeName);
-        files[targetPath] = fileContents;
+        files[targetPath] = fileContents.replace(/\s+$/g, '') + '\n';
```

Comment:
The `.replace()` call removes trailing whitespace?

Reply:
yup

Reply:
nit: There's was another PR where someone suggested using `.trim()`.  I think that would be helpful here as well since it's easier to grok.